apollo-router/src/query_planner/bridge_query_planner.rs

```diff
@@ -559,12 +559,6 @@ impl BridgeQueryPlanner {
     ) -> Result<graphql::Response, QueryPlannerError> {
         let schema = self.schema.api_schema();
         let operation = doc.get_operation(key.operation_name.as_deref())?;
-        apollo_compiler::execution::check_introspection_max_depth(&doc.executable, operation)
```

Comment:
does regular recursion limit still apply here? in other words are we opening ourselves for potential runtime errors by removing this ?

Reply:
Yes, only this new introspection-specific limit is affected. Other limits still apply, including:

* Token limit in parser
* Recursion limit in parser
* Various limits in validation (e.g. fragment nesting level)
* Router operation limits (depth, height, root fields, aliases)

Reply:
@o0Ignition0o Does that answer your inquiry?